docs/release-notes.rst

```diff
@@ -1,6 +1,58 @@
 Release Notes
 =============
 
+Restraint 0.2.1
+---------------
+
+What's New 
+~~~~~~~~~~
+* | Add ability to select `restraintd` instance by port to `restraint` commands
+  | When running commands outside of jobs on the local host,
+    some `restraint` commands require manually setting up
+    environment variables or constructing long URLs before running.
+    This can be issue if you are trying to extend the watchdog in
+    a timely fashion.  A new option was added which requires the
+    argument `--port <restraint-port-number>`. Commands affected
+    are `rstrnt-report-log`, `rstrnt-report-result`, `rstrnt-abort`, and
+    `rstrnt-watchdog`.
+* | Restore ability to specify `restraintd` port
+  | Add the `-p, --port` option back to restraint daemon and client to specify the
+    port where `restraintd` will be listening to. :bug:`1821342`
+* | Document how to remove RHTS from Jobs
+  | Added new section :ref:`rm_rhts_guide` detailing
+    substitutes for `RHTS` scripts, environment variables,
+    and testinfo.desc file and associated variables. :bug:`1802610`
+* | Setup server to listen on any local address
+  | The soup_server_listen_local() function tries to listen on both local
+    addresses, IPv4 and IPv6, and only succeeds when both are bind, even
+    when the loopback interface doesn't support one of the protocols.
+    Add rstrnt_listen_any_local() which succeeds when at least one local
+    address is bind.
```

Comment:
Fix: restraintd fails to start if both, IPv4 and IPv6, are not available on the loopback interface
In this release restraintd will not fail if it's able to listen on at least one protocol, IPv4 or IPv6, although it will still try to listen on both.